consensus/tendermint/tendermint_test.go

```diff
@@ -162,7 +162,7 @@ func TestStartRound(t *testing.T) {
 		assert.Contains(t, algo.messages.proposals[expectedHeight][expectedRound], *nodeAddr)
 		assert.Equal(t, expectedProposalMsg, algo.messages.proposals[expectedHeight][expectedRound][*nodeAddr])
 
-		assert.Equal(t, propose, algo.state.step)
+		assert.Equal(t, prevote, algo.state.step)
```

Comment:
This is because in this test, the node is the proposer, so when the rules automatically apply, it should move to prevote.